lib/billy/browsers/watir.rb

```diff
@@ -0,0 +1,55 @@
+require 'billy'
+require 'watir-webdriver'
+
+module Billy
+  module Browsers
+    class Watir < ::Watir::Browser
+
+      def initialize(name, args = {})
+        @defaults = {
```

Comment:
I don't see what this `@defaults` is helping with.  I'd prefer to see the relevant pieces within each `configure_x` method to keep it all in one place.  Also by removing this you're not instantiating `Selenium::WebDriver` unnecessarily (for non-Firefox tests).